rasa/core/policies/rule_policy.py

```diff
@@ -150,111 +150,87 @@ def predict_action_probabilities(
             return result
 
         active_form_name = tracker.active_form_name()
-        last_action_was_rejection = active_form_name and tracker.events[
-            -1
-        ] == ActionExecutionRejected(active_form_name)
+        active_form_rejected = tracker.active_loop.get("rejected")
```

Comment:
optionaL should `rejected` be a constant?

Reply:
I think this should be part of active_loop dict refactor, there are a lot of strings to be constants. Let's make in the different PR